### ISSUE TITLE
Aliasing module names with `as`

### ISSUE THREAD

**Issue by danielsoutar** [Roger-luo/FromFile.jl#39]
Simple problem: AFAIK, `FromFile.jl` does not allow aliasing module names:

```julia
# somefile.jl
module MyModule
# ...
end
# consumerfile.jl
import FromFile: @from
@from "/path/to/somefile.jl" import MyModule as MyMod
# Error looks something like this
LoadError: LoadError: LoadError: LoadError: invalid syntax import MyModule as MyMod
```

Could this be clarified in the docs? Or ideally supported, although I appreciate that from the docs it looks like this is only a package-level feature.

**Comment by patrick-kidger** [comment]
Agreed that supporting this would be good! Looks like we're hitting [this](https://github.com/Roger-luo/FromFile.jl/blob/52ee05b4f367d9a8d828c8b1b65e43a522cf30c8/src/FromFile.jl#L71) line here since  `as` isn't something we've told FromFile how to recognise.

**Comment by Roger-luo** [comment]
I think just need to forward the `as` to `using` statement then it should just work since it's supported by Julia.

**Comment by danielsoutar** [comment]
> I think just need to forward the `as` to `using` statement then it should just work since it's supported by Julia.

I didn't realise that was valid @Roger-luo, I can make the PR for that. Just to check though, does that not have different semantics compared to `import MyModule as MyMod`? That is, are the following expressions equivalent?

```julia
# Obviously not a valid line, but semantically do these expressions mean the same thing?
import MyModule as MyMod === using MyModule: MyModule as MyMod
```

If they don't then maybe a warning or similar should be conveyed to the user?

**Comment by Roger-luo** [comment]
I think this is now closed by #40
